classes/webservices/read_assignments.php

```diff
@@ -28,17 +28,18 @@
 use assign;
 use context_course;
 use context_module;
-use external_api;
-use external_function_parameters;
-use external_multiple_structure;
-use external_single_structure;
-use external_value;
-use external_warnings;
 use moodle_exception;
-use moodle_url;
 
 defined('MOODLE_INTERNAL') || die();
 
+use core_external\external_api;
```

Comment:
```suggestion
use \external_api;
```

Could you please check/confirm if using the shorter notation works correctly?